src/status_im2/contexts/wallet/collectible/view.cljs

```diff
@@ -112,6 +124,25 @@
         :subtitle      network-name
         :subtitle-type :network}]]]))
 
+(defn tabs
+  [_]
+  (let [selected-tab (reagent/atom (:id (first tabs-data)))]
+    (fn [{:keys [traits chain-id] :as _props}]
+      [:<>
+       [quo/tabs
+        {:size           32
+         :style          style/tabs
+         :scrollable?    true
+         :default-active @selected-tab
+         :data           tabs-data
```

Comment:
Maybe instead of calling them `tabs-data` we can give a name more related to what kind of tabs they are, e.g. `activity-tabs` or something like that. wdyt?

Reply:
Sure, that would be more convenient for global search!